app/models/cohort_columns/delete.rb

```diff
@@ -36,7 +36,7 @@ def display_for(user)
 
     def display_read_only(_user)
       content_tag(:a, href: pre_destroy_cohort_cohort_client_path(cohort, cohort_client), class: 'btn btn-danger btn-sm btn-icon-only', data: { loads_in_pjax_modal: true }) do
-        content_tag(:i, '', class: 'icon-cross')
+        content_tag(:i, '', class: 'icon icon-cross')
```

Comment:
@jgoley was this change intentional?  I think we have a bunch of `icon-cross` throughout the site with no extra `icon`.  (I feel like we removed that requirement at some point.

Reply:
I think I was trying to give a general icon class but I think we have css that matches "icon" at the start of the class name string. Will remove.